tests/test_model_graph.py

```diff
@@ -473,3 +481,56 @@ def test_custom_node_formatting_graphviz(simple_model):
         ]
     )
     assert body == items
+
+
+def test_none_dim_in_plate() -> None:
+    coords = {
+        "obs": range(5),
+    }
+    with pm.Model(coords=coords) as model:
+        data = pt.as_tensor_variable(
+            np.ones((5, 3)),
+            name="data",
+        )
+        pm.Deterministic("C", data, dims=("obs", None))
```

Comment:
Can you test the transposed case, with dims(None, "obs") together to confirm there are two plates?

Reply:
Sounds good. Is the non-square case still important then?